.github/workflows/jit.yml

```diff
@@ -0,0 +1,111 @@
+name: JIT
+on:
+  - pull_request
+  - push
+jobs:
+  jit:
+    # Comment this line to run JIT CI:
+    # if: false  # XXX: Uncomment before merging.
```

Comment:
Until we're ready for always building this, then rather than using comments to enable/disable, would it be useful to run builds when certain files are modified?

For example, see the `paths:` at https://github.com/python-pillow/Pillow/blob/main/.github/workflows/wheels.yml

Docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Here, we could include this workflow, `Include/internal/pycore_jit.h`, `Python/jit.c`, `Tools/jit/*` etc., or something like `**jit**`.

When needing to a build something that doesn't include those, you can temporarily comment out those restrictions, or use the `workflow_dispatch` trigger.

Reply:
Neat! Yeah, sounds like a good idea.